WeatherForecast/BL/Models/WeatherApiModel.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BL
+{
+
+    public class WeatherApiModel
+    {
+        public Coord coord { get; set; }
+        public Weather[] weather { get; set; }
+        public string _base { get; set; }
+        public Main main { get; set; }
+        public int visibility { get; set; }
+        public Wind wind { get; set; }
+        public Clouds clouds { get; set; }
+        public int dt { get; set; }
```

Comment:
properties should be meaningful and use Pascal case (E.g. public int SomeData)
If you have problem with serelization, use JsonPropertyName] attribute

Reply:
added, used JsonPropertyName